Tests/SwiftSyntaxMacrosTest/MacroSystemTests.swift

```diff
@@ -1037,4 +1058,39 @@ final class MacroSystemTests: XCTestCase {
       indentationWidth: indentationWidth
     )
   }
+
+  func testDeclsFromStringLiterals() {
+    assertMacroExpansion(
```

Comment:
As with my comment in your other PR, I would really prefer this to be three different `assertMacroExpansion` calls, each only having a single `#decl` macro expansion. It just makes debugging so much easier if the test case fails.

Test 1

```swift
struct S {
  public #decls(
    """
    static func foo() {
    print("value") }
    """,
    "struct Inner {\n\n}"
  )
}
```

Test 2

```swift
struct S {
  @attr static #decls("var value1 = 1", "typealias A = B")
}
```

Test 3

```swift
@attribute
@otherAttribute(x: 1) #decls("@moreAttibute var global = 42")
```